modules/ROOT/pages/create-community.adoc

```diff
@@ -369,25 +428,19 @@ Configure sharing settings for guests:
 .. *Field*: `Visibility` *Operator*: `equals` *Value*: `Public`
 . In *Select the users to share with*, select `[Community Name] Site Guest User`.
 . In *Select the level of access for the users*, select *Read Only*.
-. Click Save.
```

Comment:
In Bold?

Reply:
Since this is now changed to "Save.", is it correct as it is.